.devcontainer/docker-compose.yml

```diff
@@ -12,11 +12,13 @@ services:
     volumes:
       - '..:/var/www/html/wp-content/plugins/wp-graphql'
       - '../.log/app:/var/log/apache2'
+      - './mu-plugins:/var/www/html/wp-content/mu-plugins'
     env_file:
       - ../.env.dist
     environment:
-      WP_URL: http://localhost:8080/
+      WP_URL: 'https://glowing-garbanzo-ppp696r94c7v47-8080.app.github.dev/'
```

Comment:
ya, somehow this needs to be an .env var that codespaces provides. 

Right now I get this experience:

![CleanShot 2023-11-14 at 14 27 05](https://github.com/wp-graphql/wp-graphql/assets/1260765/850df195-f6c4-4450-aba6-711045fe9f41)